### PR TITLE
Use kth-style colors

### PR DESCRIPTION
**Pull request by kaj**
Use kth-style sass variables in stylesVars.scss, to get proper kth-style colors and document where our colours come from.

### CODE REVIEW

**Comment by kaj** [comment]
Note: It seems the prettier github action is out of date and don't really handle scss properly.  It works fine with my local prettier.

File: kpm/kpm-frontend/src/app.scss [hunk 1]
```diff
@@ -133,7 +133,7 @@
 
   .kpm-muted-text {
     padding-top: 0.1em;
-    color: var(--kpmPaneTextMuted);
+    color: var(--kpmPaneTextSoft);
```
Comment (jhsware) [inline kpm/kpm-frontend/src/app.scss:136]: Is there a reason for the name change? To me "muted" has more appropriate meaning than "soft".

**Review by jhsware — CHANGES_REQUESTED** [review]
The core concept is outstanding. I have a question about a name change.

File: kpm/kpm-frontend/src/app.scss (hunk 1 above)
Comment (kaj) [reply kpm/kpm-frontend/src/app.scss:136]: It's not a naming change, it's a merge.  There was both a "muted" and a "soft" variant, and I think we should only have one kind of "text with less contrast".  As the gray level is closer to the one that was called soft i kept that name (I can change that if you think muted is the better name, but it won't make the diff smaler).

Comment (jhsware) [reply kpm/kpm-frontend/src/app.scss:136]: Oh, yeah I realise I used it for different things, but that relates to our previous discussion where muted might have been a bit too light. The merge was a great idea! Personally I prefer "muted" over "soft", this is influenced by Bootstrap using `.text-muted` as a class. If you disagree, perhaps you could ask Carlos to chime in.

Comment (kaj) [reply kpm/kpm-frontend/src/app.scss:136]: I feel strongly about the level of contrast, but don't really hava an opinion about the name, so I'll change it!  :-)

**Review by jhsware — APPROVED** [review]
LGTM

**Pull request merged**